tests/integration/product_create.t

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test;
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+
+remove_all_products();
+wait_dynamic_front();
+
+my $ua = new_client();
+
+my %product_feilds = (
+	code => '200000000099',
+	lang => "en",
+	product_name => "Testttt-75ml",
+	generic_name => "Tester",
+	quantity => "75 ml",
+	link => "https://github.com/openfoodfacts/openfoodfacts-server",
+	expiration_date => "test",
+	ingredients_text => "apple, milk",
+	origin => "france",
+	serving_size => "10g",
+	packaging_text => "no",
+	".submit" => "submit"
+);
+
+create_product($ua, \%product_feilds);
```

Comment:
To create a product you have to be authenticated and you are not !
You should first use create_user (which create user and log you in, because productopener log you in at creation)

Reply:
maybe you can create a specific user for that:
```suggestion
$admin_ua = new_client();
create_user($admin_ua, {userid => "stephane", email => "stephane@test.com"})
create_product($admin_ua, \%product_feilds);
```

Reply:
That was an important step i was missing :(